ci-operator/config/openshift/assisted-service/openshift-assisted-service-cloud_hotfix_releases.yaml

```
@@ -244,7 +244,7 @@ tests:
         TEST_FUNC=test_networking
     workflow: assisted-baremetal
 - as: e2e-metal-assisted-cnv
-  optional: true
+  run_if_changed: ^(internal/operator/.*.go|internal/operators/cnv/.*|internal/operators/api/.*|internal/operators/handler/.*)$
```

very nit, but it should be 
```suggestion
  run_if_changed: ^(internal/operator/.*\.go|internal/operators/cnv/.*|internal/operators/api/.*|internal/operators/handler/.*)$
```
to match a dot with the dot character instead of every non-whitespace character

---

:facepalm:... and I forgot to change the master branch